src/core/column/column.factory.js

```diff
@@ -8,6 +13,16 @@ export default function (type, model = null) {
 	switch (type) {
 		case 'text':
 			return new TextColumn(model || TextColumn.model());
+		case 'number':
```

Comment:
I think this stuff should be better made as a map with function `register` and `get`.